ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -193,9 +193,8 @@ object Cli {
       .action( (url, config) => config.copy(authService = Some(AuthServiceJWT(JwksVerifier(url)))))
 
     opt[Unit]("sortable-contract-ids")
-      .hidden()
```

Comment:
Please keep it hidden, as it is more a flag for us to be able to toggle it, but not intended to be used by end users.

Reply:
done.